### PR TITLE
Documented how to recover an etcd cluster where one node got corrupt

### PR DESCRIPTION
**Pull request by arianvp**
(fill in your PR description)

## Checklist:

Please tick the following before making your PR:

* [ ] I ran `make` on this branch to build the docs and there are **no WARNINGs**.
* [x] After running `make`, I looked at the output by opening build/html/index.html in a browser and I'm satisfied with the rendering.

### CODE REVIEW

File: source/how-to/administrate/etcd.rst [hunk 1]
```diff
@@ -56,6 +56,15 @@ On each server one by one:
 6. Do the same on the next server.
 
 
+Backing up and restoring
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Though as long as quorum is maintained in etcd there will be no dataloss, it is still good to prepare 
+for the worst. If a disaster takes out all nodes, then you might want to restore from an old backup.
+
+Luckily, etcd can take periodic snapshots of your cluster and these can be used in cases of disaster recovery.
```
Comment (arianvp) [inline source/how-to/administrate/etcd.rst]: This is not related to what I was actually fixing, I just thought it was useful to have this section. Should expand on it later

**Comment by jschaul** [comment]
Please check the above checkboxes and rebase for the `source` -> `src` path change.

**Comment by arianvp** [comment]
There were a few warnings:
```
src/how-to/install/ansible-dependencies.rst:9: WARNING: duplicate label ansible-deps-option-1, other instance in /mnt/src/how-to/install/ansible-dependencies.rst
src/how-to/install/ansible-dependencies.rst:43: WARNING: duplicate label ansible-deps-option-2, other instance in /mnt/src/how-to/install/ansible-dependencies.rst
src/how-to/install/ansible-dependencies.rst:9: WARNING: duplicate label ansible-deps-option-1, other instance in /mnt/src/how-to/install/kubernetes.rst
src/how-to/install/ansible-dependencies.rst:43: WARNING: duplicate label ansible-deps-option-2, other instance in /mnt/src/how-to/install/kubernetes.rst
```

**Comment by jschaul** [comment]
> There were a few warnings, but not in places I touched:
> 
> ```
> src/how-to/install/ansible-dependencies.rst:9: WARNING: duplicate label ansible-deps-option-1, other instance in /mnt/src/how-to/install/ansible-dependencies.rst
> src/how-to/install/ansible-dependencies.rst:43: WARNING: duplicate label ansible-deps-option-2, other instance in /mnt/src/how-to/install/ansible-dependencies.rst
> src/how-to/install/ansible-dependencies.rst:9: WARNING: duplicate label ansible-deps-option-1, other instance in /mnt/src/how-to/install/kubernetes.rst
> src/how-to/install/ansible-dependencies.rst:43: WARNING: duplicate label ansible-deps-option-2, other instance in /mnt/src/how-to/install/kubernetes.rst
> ```
> 
> Apart from that, this is ready to merge. I prefer to split this up later from Joe's suggestions about How-To vs Explanation.

Hm interesting, those warnings are not related to your change so feel free to merge. (I wonder why I don't see those warnings though...)

**Pull request merged**